### DISCUSSION TITLE
semver tags for docker images ?

### DISCUSSION THREAD

**Discussion by euri10** [jesec/rtorrent#3] [Ideas]
It would be awesome to have semver tags for the docker images (https://hub.docker.com/r/jesec/rtorrent/tags?page=1&ordering=last_updated)

I'm using renovate bot to check my images and that would help immensely being up to date with latest changes :)

thanks a lot, this and flood rocks !

**Comment by jesec** [comment]
Good idea. Though, unfortunately, that is not possible at this moment. Currently the project simply doesn’t follow semver versioning practices. Main version will stay at `0.9.8` for the time being, and backward incompatible changes are not expected at this stage.
